packages/protocol/test/L2/TaikoL2AdjustExcess.t.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../TaikoTest.sol";
+import "../../contracts/L1/TaikoData.sol";
+
+contract TestTaikoL2AdjustExcess is TaikoTest {
+    using SafeCast for uint256;
+
+    // Initial salt for semi-random generation
+    uint256 salt = 2_195_684_615_435_261_315_311;
+    // same as `block_gas_limit` in foundry.toml
+    uint32 public constant BLOCK_GAS_LIMIT = 30_000_000;
+
+    address public addressManager;
+    TaikoL2EIP1559Configurable public L2;
+
+    function setUp() public {
+        addressManager = deployProxy({
+            name: "address_manager",
+            impl: address(new AddressManager()),
+            data: abi.encodeCall(AddressManager.init, (address(0)))
+        });
+
+        SignalService ss = SignalService(
+            deployProxy({
+                name: "signal_service",
+                impl: address(new SignalService()),
+                data: abi.encodeCall(SignalService.init, (address(0), addressManager)),
+                registerTo: addressManager
+            })
+        );
+
+        uint64 gasExcess = 20_000_000_000;
```

Comment:
It's because this value is too large. We should use 0 as the initial gas excess.